rosdep/base.yaml

```diff
@@ -3999,10 +3999,22 @@ libv4l-dev:
       packages: [v4l-utils]
   ubuntu: [libv4l-dev]
 libvlc:
-  debian: [libvlc5, vlc-nox]
+  debian:
+    '*': [libvlc5, vlc-bin]
+    buster: [libvlc5, vlc-nox]
```

Comment:
Debian Buster came after Debian Stretch, so shouldn't this line be removed?

Reply:
You are right, I got my Debian version names mixed up.